src/components/common/BoardSection.tsx

```diff
@@ -7,8 +7,8 @@ import ViewListIcon from '@/assets/ViewListIcon';
 
 interface BoardSection {
   title: keyof typeof ICON;
-  content?: ReactNode;
-  children?: ReactNode;
+  content: ReactNode;
+  mode?: 'month' | 'week';
```

Comment:
```suggestion
  mode?: 'month' | 'week';
  content?: ReactNode;
  children?: ReactNode;
```
기존 작업 부분이 삭제돼서 이렇게 바꿔주세요!